app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryFragment.kt

```diff
@@ -83,7 +83,7 @@ class TokenHistoryFragment :
         totalTextView.text = tokenForHistory.getFormattedTotal(includeSymbol = true)
         usdTotalTextView.text = tokenForHistory.getFormattedUsdTotal()
 
-        refreshLayout.setOnRefreshListener { presenter.refresh() }
+        refreshLayout.setOnRefreshListener { presenter.retry() }
```

Comment:
what should presenter retry? RetryLoad? retry parsing? I suggest to choose more specific and explicit name for this method